client/document/404.jsx

```diff
@@ -13,12 +15,18 @@ import EmptyContent from 'components/empty-content';
 import { chunkCssLinks } from './utils';
 
 function NotFound( { faviconURL, entrypoint } ) {
+	const theme = config( 'theme' );
+
 	return (
 		<html lang="en">
 			<Head faviconURL={ faviconURL } cdn={ '//s1.wp.com' }>
 				{ chunkCssLinks( entrypoint ) }
 			</Head>
-			<body>
+			<body
+				className={ classNames( {
+					[ 'theme-' + theme ]: theme,
```

Comment:
Should this also happen for 500 errors?

Reply:
yeah I should add it there as well. :+1: